apps/src/craft/aquatic/craft.js

```diff
@@ -374,7 +375,7 @@ Craft.initializeAppLevel = function (levelConfig) {
     entities: levelConfig.entities,
     playerStartPosition: levelConfig.playerStartPosition,
     playerStartDirection: levelConfig.playerStartDirection,
-    playerName: Craft.getCurrentCharacter(),
+    playerName: Craft.getCurrentCharacter() + 'Aquatic',
```

Comment:
is there a specific reason this is added here an not in `getCurrentCharacter` itself?

Reply:
I think because we still key the skin avatars off the base name:

https://github.com/code-dot-org/code-dot-org/blob/ca82a02b457464b1c7ec12212c5b034b61810a38/apps/src/craft/aquatic/craft.js#L199-L203

This is following the pattern in the Designer & Hero levels:

https://github.com/code-dot-org/code-dot-org/blob/ca82a02b457464b1c7ec12212c5b034b61810a38/apps/src/craft/designer/craft.js#L529